src/test/java/org/openapitools/jackson/nullable/JsonNullableValueExtractorTest.java

```diff
@@ -17,15 +19,23 @@
 
 
 public class JsonNullableValueExtractorTest {
-
+    private ValidatorFactory factory;
     private Validator validator;
 
     @Before
     public void setUp() {
-        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        factory = Validation.byDefaultProvider()
+                .configure()
+                .messageInterpolator(new ParameterMessageInterpolator())
+                .buildValidatorFactory();
         validator = factory.getValidator();
     }
 
+    @After
+    public void tearDown() {
+        factory.close();
```

Comment:
My IDE warned about the factory not being closed

Reply:
Probably needs a null- check.